### ISSUE TITLE
Mapped SMES do funny things when renamed

### ISSUE THREAD

**Issue by TheLordME** [Citadel-Station-13/Citadel-Station-13-RP#2619]
#### Brief description of the bug
Issue: set up a second SMES for station power and named the original 'power - main' to Power - main 1 and the second to 'power - main 2'.  RCON  shows those two, as well as 4 instances of 'power - main'  All seem to be outputting.

#### What you expected to happen
That there is only existing SMES shown in RCON

#### What actually happened
4 instances of a no longer existing SMES appeard in RCON

#### Steps to reproduce
- Built second SMES in Engine SMES room
- Rename Power - Main SMES to Power - Main 1
- Name second SMES Power - Main 2
- look at RCON and notice  multiple Power - Main SMES doing stuff

#### Code Revision
- Dont know recent to creation of the issue.
 
#### Anything else you may wish to add:
- Didnt find this issue any questions go to lupint#1862 on discord

**Comment by Lupinius** [comment]
Specifics:  First thing done once I arrived to engineering was construct the second SMES, naming it to 'Power - Main 2'  did not activate the input or output on it,  just set the value to 1800 for input and 1200 for output.  Left the 'Power- Main' alone.
 Went on to set up the engine itself, piping, adding phoron, inserting rods.   Once it was generating power I went back to monitor the 'Power - main' SMES, waiting for it to fully discharge before activating the newer SMES, To speed up the discharge of main, I went to RCON to activate the various inputs for the stations SMES's, that's when I noticed there were extra Main SMES's in addition to the one I added, (named Power - Main 2).  Once The original SMES was discharged and I activated the second SMES, I unscrewed the original SMES to upgrade it's coils. Once done and setting it to mirror the second SMES, then renaming it 'Power - Main 1,  Checking RCON again, it now showed the 4 Power - Main  listed, fully charged, input was off but output was on for each. the outputs were 0 except for one which was fluctuating around 18 kW. that same one also showed a 95 percent charge and stayed that way throughout the round. The now renamed two SMES's that I worked on behaved normally, Main 1 slowly charging and fully outputting to the station, Main 2 taking up any slack for power output.
